desktop/src/store/Shift/endpoints.js

```diff
@@ -22,7 +22,7 @@ export const put = (shift) => {
 
 export const getCurrentShift = (employeeId) => {
   return axios.get(
-    `/employees/${employeeId}/shifts?filter[limit]=1&filter[order]=id DESC`,
+    `/employees/${employeeId}/shifts?filter[limit]=1&filter[order]=clock_out_date`
```

Comment:
I would find all references to this and test that those operations still work just to make sure.

Reply:
You could rename this since `current` is vague.

Reply:
Maybe `getHalfShiftOrNull`.

Reply:
Done, renamed all instances!